js/src/util/scrollbar.js

```diff
@@ -8,7 +8,7 @@
 import SelectorEngine from '../dom/selector-engine'
 import Manipulator from '../dom/manipulator'
 
-const SELECTOR_FIXED_CONTENT = '.fixed-top, .fixed-bottom, .is-fixed'
+const SELECTOR_FIXED_CONTENT = '.fixed-top, .fixed-bottom, .is-fixed, .sticky-top'
```

Comment:
I wish it would be me :P

In this PR  I just remove the copied the code, out of modal.
It was written like this. I am trying to finish this and a new issue waits #33580 (afraid of how many tests need to tweak)

Reply:
Oh I see. Figured you realized a bug and fixed it that way. Nvm then.

Reply:
One thing to mention maybe. We got reponsive sticky classes too 😄